### PR TITLE
[Installation] Add instructions for snap

### PR DESCRIPTION
**Pull request by AdrikaPanwar**
Hi, I found that in your installation.md there is no instructions to install yt-dlp using snap so I add it, I will be happy if you will review it and merge it

### CODE REVIEW

**Comment by bashonly** [comment]
The yt-dlp version in the snapcraft repository is `2024.08.06`

I'd rather not add instructions for how to install outdated yt-dlp

The other 3rd party package managers that we provide instructions for are at most a few days behind

**Comment by bashonly** [comment]
Ah, nevermind I see they have an edge version available too. How can we be sure the user is installing the edge version?

File: Installation.md [hunk 1]
```diff
@@ -118,6 +118,18 @@ Your system's package manager will now automatically download the correct depend
 sudo apt update
 sudo apt install yt-dlp
 ```
+
+### [snap](https://snapcraft.io/yt-dlp)
+
+You can install yt-dlp using snap on Linux:
+```bash
+sudo snap install yt-dlp
```
Comment (grqz) [inline Installation.md:126]:
```suggestion
sudo snap install yt-dlp --channel=latest/edge
```

**Review by bashonly — APPROVED** [review]

**Review by AdrikaPanwar — COMMENTED** [review]
Thankyou for resolving this

File: Installation.md [hunk 2]
```diff
@@ -118,6 +118,19 @@ Your system's package manager will now automatically download the correct depend
 sudo apt update
 sudo apt install yt-dlp
 ```
+
+### [Snap](https://snapcraft.io/yt-dlp)
+
+You can install yt-dlp on Linux using Snap:
+```bash
+sudo snap install yt-dlp --channel=latest/edge
```
Comment (bashonly) [inline Installation.md:126]:
```suggestion
sudo snap install --edge yt-dlp
```

File: Installation.md [hunk 3]
```diff
@@ -118,6 +118,19 @@ Your system's package manager will now automatically download the correct depend
...
+```
+
+To manually update, run:
+```bash
+sudo snap refresh yt-dlp --channel=latest/edge
```
Comment (bashonly) [inline Installation.md:131]:
```suggestion
sudo snap refresh --edge yt-dlp
```

**Review by bashonly — APPROVED** [review]

**Pull request merged**